### PR TITLE
fix  getUserId issue // To be merged after 2023.3.4 go prod 100%

### PR DESCRIPTION
**Pull request by meladRaouf**
I added the fix here so that we don't miss it.

### CODE REVIEW

**Review by alex-vt — CHANGES_REQUESTED** [review]
The unit test probably should be ported too (replace `GROUP` with `Partitioning` there).

**Review by alex-vt — APPROVED** [review]

File: infra/event-sync/src/test/java/com/simprints/infra/eventsync/status/down/EventDownSyncScopeRepositoryTest.kt
Comment (alex-vt) [inline infra/event-sync/src/test/java/com/simprints/infra/eventsync/status/down/EventDownSyncScopeRepositoryTest.kt:1]: For reference: https://github.com/Simprints/Android-Simprints-ID/pull/538/files#diff-5ef858eb5a58e8a3c5f41eba2263985fd80e986141fb2252b1767341a65fd6e1

**Review by alex-vt — CHANGES_REQUESTED** [review]

**Review by alex-vt — APPROVED** [review]

**Review by BurningAXE — APPROVED** [review]

**Review by alexandr-simprints — APPROVED** [review]

**Comment by luhmirin-s** [comment]
I think this is redundant since we added a fix for MS-201 into the next release.

**Comment by meladRaouf** [comment]
This fix was already added to  2023.4.1

**Pull request closed without merging**